### PR TITLE
Task#2: Create UI for products - cards

### LINKED ISSUE #50: User story#1: Products: UI Design
- Task#2  Create filter interface and logo icon for manufacturer.

- [x] Full 3 filters

- [x] List logo for manufacturer

- Task#3 Create card details tags for each product

- [x] Must have image at the top
- [x] Label for screen pixel size and type
- [x] Price of the product

- Task#4 Edit the style for the interface

- [x] User can hover the filter button
- [x] Users can hover the logo manufacturer.

- Task#5 Testing the features and display 

- [x] Check if there are enough products

**_Task2, Task3, task4, task 5 One in Task#2 Create UI for products - cards #51_**

- [x] #52
- [x] #58
- [x] #61

### PR DESCRIPTION
**Pull request by TheHienDuong**
- #49 
  - #50 
## OVERVIEW

> -  Create filter interface and logo icon for manufacturer
> - Create card details tags for each product
> - Edit the style for the interface

## DEMO
image

### CODE REVIEW

File: practice-02/styles/index.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+.filter-item__title {
+  background-color: var(--light);
+  border: 1px solid var(--white);
+  border-radius: 4px;
+  cursor: pointer;
+  padding: 8px 10px;
+}
+
+.filter-item__title span {
```
Comment (phattruong-agilityio) [inline practice-02/styles/index.css]: always using class. it's have better performance than css by tag

File: practice-02/styles/index.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+  height: 20px;
+  margin-right: 5px;
+}
+
+/* CategoryApp */
+.box-sort {
+  margin: 10px 0;
+}
+
+.productsbox {
```
Comment (phattruong-agilityio) [inline practice-02/styles/index.css]: `productsbox` please check it

File: practice-02/styles/index.css [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+  display: block;
+  overflow: hidden;
+  width: 100%;
+}
+
+.listproduct {
```
Comment (phattruong-agilityio) [inline practice-02/styles/index.css]: `listproduct` please help check

File: practice-02/styles/index.css [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+.listproduct .item-img .thumb {
+  margin-bottom: 0;
+  margin-top: 8px;
+}
+
+.listproduct .thumb {
+  max-width: 100%;
+}
+
+.listproduct h3 {
```
Comment (phattruong-agilityio) [inline practice-02/styles/index.css]: the same as span

**Comment by phattruong-agilityio** [comment]
Please using arial font

File: practice-02/styles/index.css [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+}
+
+.listproduct {
+  display: grid;
+  grid-template-columns: repeat(5, minmax(0, 1fr));
+  border-left: 1px solid var(--light);
+  border-top: 1px solid var(--light);
+}
+
+.listproduct .item {
```
Comment (phattruong-agilityio) [inline practice-02/styles/index.css]: How about this one

File: practice-02/styles/index.css (hunk 3 above)
Comment (phattruong-agilityio) [reply practice-02/styles/index.css]: Did you check it?

File: practice-02/styles/index.css (hunk 5 above)
Comment (TheHienDuong) [reply practice-02/styles/index.css]: I have checked all classes like that

File: practice-02/styles/index.css (hunk 3 above)
Comment (TheHienDuong) [reply practice-02/styles/index.css]: I have checked all classes like that

File: practice-02/styles/index.css (hunk 2 above)
Comment (TheHienDuong) [reply practice-02/styles/index.css]: Yes, I checked and changed the name

File: practice-02/.gitignore [hunk 6]
```diff
@@ -1 +1,3 @@
 node_modules
+
+package-lock.json
```
Comment (phattruong-agilityio) [inline practice-02/.gitignore]: Remove this

Comment (phattruong-agilityio) [reply practice-02/.gitignore]: package-lock is always keep

File: practice-02/src/index.html [hunk 7]
```diff
@@ -1,12 +1,108 @@
...
+    <main class="container">
+        <section id="box-filter">
+            <!-- Filter -->
+            <div class="block-filter-main">
+                <div class="filter-total">
+                    <div class="filter-item__title">Filter</div>
+                </div>
+                <div class="filter-item">
+                    <div class="filter-item__title">
+                        <span class="title"> Manufacturer </span>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: Why have space here

File: practice-02/src/index.html [hunk 8]
```diff
@@ -1,12 +1,108 @@
...
+                    </div>
+                </div>
+                <div class="filter-item">
+                    <div class="filter-item__title">
+                        <span class="title"> Price </span>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: Why have space here

File: practice-02/src/index.html [hunk 9]
```diff
@@ -1,12 +1,108 @@
...
+                            <div class="item-img">
+                                <img class="thumb"
+                                    src="https://cdn.tgdd.vn/Products/Images/42/274018/samsung-galaxy-a24-black-thumb-600x600.jpg" alt="Samsung Galaxy A24" />
+                            </div>
+                            <h3 class="item-name">Samsung Galaxy A24</h3>
+                            <div class="item-compare">
+                                <span class="text">6.5"</span>
+                                <span class="text">Full HD+</span>
+                            </div>
+                            <strong class="price">6.090.000₫</strong>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: using ₫ in HTML decode

File: practice-02/src/index.html (hunk 8 above)
Comment (TheHienDuong) [reply practice-02/src/index.html]: yes i removed the space around the text

File: practice-02/src/index.html (hunk 9 above)
Comment (TheHienDuong) [reply practice-02/src/index.html:98]: yes i removed the space around the text

File: practice-02/src/index.html (hunk 7 above)
Comment (TheHienDuong) [reply practice-02/src/index.html]: yes i removed the space around the text

**Pull request merged**